src/main/java/net/atos/zac/util/MediaTypes.kt

```diff
@@ -0,0 +1,321 @@
+/*
+ * SPDX-FileCopyrightText: 2024 Lifely
+ * SPDX-License-Identifier: EUPL-1.2+
+ */
+package net.atos.zac.util
+
+/**
+ * List of media types used by the application.
+ *
+ * See [Media Types](https://www.iana.org/assignments/media-types/media-types.xhtml) of IANA.
+ *
+ * Thanks goes out to [Youngbin Kim](https://gist.github.com/retheviper) for providing the original code:
+ * https://gist.github.com/retheviper/9b4e28f66b354d9f706e43d399100676
+ */
+object MediaTypes {
+    /** Hypertext Application Language (HAL) for JSON */
+    const val MEDIA_TYPE_HAL_JSON = "application/hal+json"
+
+    /** Standardized format (RFC 7807) for representing error responses in HTTP APIs */
+    const val MEDIA_TYPE_PROBLEM_JSON = "application/problem+json"
+
+    enum class Application(val extensions: Array<String>, val mediaType: String) {
+        /** AbiWord document */
+        ABI_WORD(arrayOf(".abw"), "application/x-abiword"),
+
+        /** Archive document (multiple files embedded) */
+        ARCHIVE(arrayOf(".arc"), "application/x-freearc"),
+
+        /** Atom Documents */
+        ATOM_XML(arrayOf(".atom"), "application/atom+xml"),
+
+        /** Atom Category Documents */
+        ATOMCAT_XML(arrayOf(".atomcat"), "application/atomcat+xml"),
+
+        /** Amazon Kindle eBook format */
+        AMAZON_KINDLE(arrayOf(".azw"), "application/vnd.amazon.ebook"),
+
+        /** Apple Installer Package */
+        APPLE_INSTALLER(arrayOf(".mpkg"), "application/vnd.apple.installer+xml"),
+
+        /** C-Shell script */
+        CSH(arrayOf(".csh"), "application/x-csh"),
+
+        /** ECMAScript */
+        ECMASCRIPT(arrayOf(".es"), "application/ecmascript"),
+
+        /** Electronic publication (EPUB) */
+        EPUB(arrayOf(".epub"), "application/epub+zip"),
+
+        /** GZip Compressed Archive */
+        GZIP(arrayOf(".gz"), "application/gzip"),
+
+        /** Java Archive (JAR) */
+        JAVA_ARCHIVE(arrayOf(".jar"), "application/java-archive"),
+
+        /** JavaScript */
+        JAVASCRIPT(arrayOf(".js"), "application/javascript"),
+
+        /** JSON format */
+        JSON(arrayOf(".json"), "application/json"),
```

Comment:
Can we actually use only this one in ZAC and populate the enum with what's available in JAX-RS?

Reply:
Not really. The issue is that e.g. `@Produces` annotations require a constant. So these enums cannot be used there.. My thought now is: use the constants defined in JAX-RS and for other media types which require constants define them ourselves. I think that's ok.